management-api-server/src/test/java/com/datastax/mgmtapi/NonDestructiveOpsIT.java

```diff
@@ -1080,4 +1097,14 @@ private Pair<Integer, String> responseAsCodeAndBody(FullHttpResponse r) {
 
     return Pair.of(copy.status().code(), null);
   }
+
+  private int getNumTokenRanges() {
+    if (this.version.startsWith("3")) {
+      return 256;
+    }
+    if (this.version.startsWith("dse-68")) {
+      return 1;
+    }
+    return 16;
```

Comment:
Suggestion: it might be nice to have this explicitly check for Cassandra 4.x so that it is clear what we're trying to support.